<?xml version="1.0"?>
<launch>

<node name="lvr_fusion" pkg="lvr_ros" type="lvr_fusion_node" output="screen">
	<param name="verbose_lvr" value="false"/>
	<param name="publish_constantly" value="false"/>

	<param name="input_mesh_topic" value="/reconstruction_mesh"/>
	<param name="output_mesh_topic" value="/fusion_mesh"/>
	<!--param name="output_textures_topic" value="/fusion_textures"/-->

	<param name="type" value="remote"/>
	<param name="distance_threshold" value="0.05"/>
</node>

</launch>



